lib/danger/ci_source/bitrise.rb

```diff
@@ -0,0 +1,37 @@
+# http://devcenter.bitrise.io/docs/available-environment-variables
+
+module Danger
+  # ### CI Setup
+  #
+  # Add a script step to your workflow:
+  # ``` yml
+  # - script@1.1.2:
+  #     inputs:
+  #    - content: |-
+  #        bundle install
+  #        bundle exec danger
+  # ```
+  # ### Token Setup
+  #
+  # Add the `DANGER_GITHUB_API_TOKEN` to your workflow's Secret Env Vars
+  #
+  class Bitrise < CI
+    def self.validates_as_ci?(env)
+      env.key? "BITRISE_BUILD_URL"
```

Comment:
or you could use `BITRISE_IO` - set to `"true"` on bitrise.io VMs ; although that's true for our Docker images as well, if you'd use it locally https://github.com/bitrise-docker/bitrise-base/blob/master/Dockerfile#L15